### ISSUE TITLE
Issue ID: DAC_Unstructured_Lists2

### ISSUE THREAD

**Issue by VDuczekW** [guardian/dotcom-rendering#5079]
Report Link: https://drive.google.com/file/d/1vp5M4hxnNBZkh3jronwR8xorzvylb584/view?usp=sharing

Full issue list: https://docs.google.com/spreadsheets/d/1vWQtZfD49uq7uNCNELuYN_HcIj-BKiE4t4Hwda3tJaQ/edit#gid=0

**Comment by bryophyta** [comment]
>**Note**
>This is an automatically-generated comment. Feel free to edit the content, but please don't edit this 'note' block.
 
# Intro to report

Copying the intro here to aid discoverability of the report's content:
```
Issue ID: DAC_Unstructured_Lists2
URL: https://www.theguardian.com/football/ng-interactive/2021/sep/07/david-squires-onwenger-revenge-fifa-world-cup-qualifiers-cr7-england-hungary-brazil-football
Page title: David Squires on … Wenger’s revenge, CR7’s shirt and love conquering hate |
Football | The Guardian
Journey: 1.16
Screenshot:

For a list to be valid, it must have both parent and child elements. Screen readers notify
users when they come to a list and tell them how many items are in a list. Announcing the
number of items in a list and the current list item helps listeners know what they are
listening to, and what to expect as they listen to it.
In this instance the <ol> has a role="tabpanel" attribute which effects the list markup, and
strips the structure of the list. The list items cannot inform the screen reader user that they
are listening to a list when there is no parent indicating the presence of a list and the type of
list.
```
